src/organisms/navigation/hooks/useOnTabsResize.js

```diff
@@ -1,6 +1,8 @@
 import { useCallback, useRef } from "react"
 
 export default (ref, target, callback, deps) => {
+  const prevValuesRef = useRef({})
```

Comment:
Could set outside the initial value.
`const initialRect = {prevContainerRight: -1, prevTabWidth: -1}`